scripts/test-sim-benchmark.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -ueo pipefail
+
+# Purpose: This script executes uplink upload and download benchmark tests against storj-sim.
+# Setup: Setup and run storj-sim. Remove existing uplink configs.
+# Usage: from the root of storj dir, run ./scripts/test-sim-benchmark.sh
+
+# create an api key that the uplink will use
+regEndpoint=http://127.0.0.1:10002/registrationToken/?projectsLimit=1;
+endpoint=http://127.0.0.1:10002/api/graphql/v0;
+satellitePublicGRPC=http://127.0.0.1:10000;
+
+email=`date +%s`@email.com;
```

Comment:
`date %+s` has a granularity of 1 second, of course. With that long of an interval, it is practically certain that situations will arise where there are multiple identical values generated. Does that matter here?

If it doesn't matter at all, I suggest using a static value to avoid people assuming in the future that it is unique. If it does matter, you might be able to get something better by abusing `mktemp` or something of that nature.